modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -223,16 +217,7 @@ export default class NestedTabs extends Base {
 		this.elements.$tabTitles.on( this.getTabEvents() );
 		this.elements.$headingContainer.on( this.getHeadingEvents() );
 
-		const settingsObject = {
-			element: this.elements.$headingContainer[ 0 ],
-			direction: this.getTabsDirection(),
-			justifyCSSVariable: '--n-tabs-heading-justify-content',
-			horizontalScrollStatus: this.getHorizontalScrollSetting(),
-		};
-
-		this.resizeListenerNestedTabs = setHorizontalScrollAlignment.bind( this, settingsObject );
-		elementorFrontend.elements.$window.on( 'resize', this.resizeListenerNestedTabs );
```

Comment:
The resize update didn't work correctly in 3.22 either, but the re-rendering of the widget triggered the scrolling update.